test/parallel/test-fs-read-stream-err.js

```diff
@@ -1,43 +1,43 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var fs = require('fs');
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
 
-var stream = fs.createReadStream(__filename, {
+const stream = fs.createReadStream(__filename, {
   bufferSize: 64
 });
-var err = new Error('BAM');
+const err = new Error('BAM');
 
 stream.on('error', common.mustCall(function errorHandler(err_) {
   console.error('error event');
-  process.nextTick(function() {
-    assert.equal(stream.fd, null);
-    assert.equal(err_, err);
+  process.nextTick(() => {
```

Comment:
I see your thinking, but think its too subtle, function names don't matter, particularly not in tests. Better change them all.